### PR TITLE
fix deviceTwin device update fields

### PR DESCRIPTION
**Pull request by TianTianBigWang**
Signed-off-by: TianTianBigWang <zilong.wang@daocloud.io>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
The first err is not processed

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
/lgtm
but need to fix test error

**Comment by gy95** [comment]
maybe too much modification, IMHO, we just fix a little, and maybe ut just need a little fix, now ut fix too much.

File: edge/pkg/devicetwin/dtmanager/device_test.go [hunk 1]
```diff
@@ -237,14 +244,13 @@ func TestDealDeviceStateUpdate(t *testing.T) {
 			updateReturnInt:  int64(1),
 			updateReturnErr:  nil,
 			queryTableReturn: querySeterMock,
-			times:            2,
```
Comment (gy95) [inline edge/pkg/devicetwin/dtmanager/device_test.go:240]: I try to fix this from 2 to 1, and ut passed.

Comment (TianTianBigWang) [reply edge/pkg/devicetwin/dtmanager/device_test.go:240]: 😲seem that you're right

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**